From 495fc39665a25e43f732931642267ade14a981ba Mon Sep 17 00:00:00 2001
From: James Kosin <jkosin@apache.org>
Date: Fri, 20 Jan 2012 04:20:25 +0000
Subject: [PATCH] OPENNLP-367: CONLL 03 data for German & English must be in
 UTF-8.  I tried many combinations of ISO and the German doesn't come out
 correctly without UTF-8.  I'm hoping this fixes the last of the encoding
 issues for this round.

git-svn-id: https://svn.apache.org/repos/asf/incubator/opennlp/trunk@1233753 13f79535-47bb-0310-9956-ffa450edef68
---
 .../java/opennlp/tools/formats/Conll03NameSampleStream.java   | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/opennlp-tools/src/main/java/opennlp/tools/formats/Conll03NameSampleStream.java b/opennlp-tools/src/main/java/opennlp/tools/formats/Conll03NameSampleStream.java
index b4f6c8328..c07a64a75 100644
--- a/opennlp-tools/src/main/java/opennlp/tools/formats/Conll03NameSampleStream.java
+++ b/opennlp-tools/src/main/java/opennlp/tools/formats/Conll03NameSampleStream.java
@@ -67,8 +67,8 @@ public Conll03NameSampleStream(LANGUAGE lang, InputStream in, int types) {
 
     this.lang = lang;
     try {
-      this.lineStream = new PlainTextByLineStream(in, "ISO-8859-1");
-      System.setOut(new PrintStream(System.out, true, "ISO-8859-1"));
+      this.lineStream = new PlainTextByLineStream(in, "UTF-8");
+      System.setOut(new PrintStream(System.out, true, "UTF-8"));
     } catch (UnsupportedEncodingException e) {
       // UTF-8 is available on all JVMs, will never happen
       throw new IllegalStateException(e);
